src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -148,6 +153,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        if (!method_exists($this->extractor, 'blank') && true === $input->getOption('blank')) {
+            $errorIo->error('The --blank option is not supported by this extractor. It is available in symfony/translation 6.1 and symfony/twig-bridge 6.1.');
+
+            return 1;
+        }
+
```

Comment:
I added the error message up here because other checks like this also live here. And I don't need to pass the OutputInterface down to the method. Seems cleaner this way.

Not sure about the wording of the message. Anything you want to change or add?